### PR TITLE
scanResponse is never used

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

File: repm/types.go [hunk 1]
```diff
@@ -47,11 +47,6 @@ type scanItem struct {
 	Value jsnoms.Value `json:"value"`
 }
 
-type scanResponse struct {
```
Comment (aboodman) [inline repm/types.go:50]: It's nice for documentation purposes. Can we do this the other way - adapt this to be the thing scan actually returns and then use it?

**Comment by arv** [comment]
PTAL

File: repm/types.go [hunk 2]
```diff
@@ -42,16 +42,13 @@ type scanRequest struct {
 	db.ScanOptions
 }
 
+type scanResponse = []db.ScanItem
```
Comment (aboodman) [inline repm/types.go:45]: Type aliases are disrecommended for everyday use. Why doesn't `type scanResponse []db.ScanItem` work?

Comment (arv) [reply repm/types.go:45]: I didn't know about the distinction.

**Pull request merged**